packages/angular/projects/ui-angular/src/lib/components/authenticator/components/authenticator/authenticator.component.ts

```diff
@@ -66,15 +67,17 @@ export class AuthenticatorComponent
       formFields,
     } = this;
 
-    this.unsubscribeHub = listenToAuthHub((event) => {
+    const { authService } = this.authenticator;
+
+    this.unsubscribeHub = listenToAuthHub(authService, (data, service) => {
+      defaultAuthHubHandler(data, service);
```

Comment:
So this is more of a decorator pattern now -- use the default auth hub handler, then angular-specific custom handling is appended.

Note that this is Angular only.